packages/suite/src/components/wallet/Fees/CustomFee.tsx

```diff
@@ -157,14 +164,20 @@ export const CustomFee = <TFieldValues extends FormState>({
     return (
         <motion.div
             initial={{ opacity: 0, height: 0, marginTop: 0 }}
-            animate={{ opacity: 1, height: 'auto', marginTop: 20 }}
+            animate={{ opacity: 1, height: 'auto', marginTop: spacingsPx.sm }}
             exit={{ opacity: 0, height: 0, marginTop: 0 }}
             transition={{
                 opacity: { duration: 0.15, ease: motionEasing.transition },
                 height: { duration: 0.2, ease: motionEasing.transition },
                 marginTop: { duration: 0.25, ease: motionEasing.transition },
             }}
         >
+            <StyledWarning withIcon variant="warning">
```

Comment:
We try to avoid overriding DS (design system) components and in the future it's going to be forbidden. 

Please use this way:
```tsx
<Warning margin={{ bottom: spacings.xs }}>
```

Reply:
Thank you, changed